data/Chamberezigbo.json

```
@@ -1,10 +1,11 @@
 {
   "name": "Chamberlain Ezigbo",
+  "type": "personal",
   "bio": "JavaScript Engineer | Full-Stack Developer",
   "links": [
     {
       "name": "My portfolio site",
-      "url": "https://wondrous-churros-f6aea2.netlify.app",
+      "url": "chamberline.netlify.app",
```

```suggestion
      "url": "https://chamberline.netlify.app",
```
I think this should have a full URL link